docs/socorro_test_checklist.rst

```diff
@@ -0,0 +1,123 @@
+======================
+Socorro Test Checklist
+======================
+
+This is a high-level system-wide checklist for making sure Socorro is working
+correctly in a specific environment. It's helpful after we've made a significant
+change.
+
+.. contents::
+
+
+Collector
+=========
+
+Is the collector web process handling incoming crashes?
+
+    * Log into collector node and watch the collector logs for errors.
+
+Is the collector crashmover process saving crashes to S3? ES? Postgres?
+RabbitMQ?
+
+    * Log into collector node and watch the crashmover logs for errors.
+
+      FIXME: List log paths.
+
+    * Check datadog ``crashmover.save_raw_crash`` for the appropriate
+      environment.
+
+      :prod: it's on the Socorro Prod dashboard
+      :stage: ?
+  
+    * Submit a crash. Verify raw crash made it to S3. See these
+      docs:
+      http://socorro.readthedocs.io/en/latest/configuring-socorro.html?highlight=curl#test-collection-and-processing
+   
+
+Processor
+=========
+
+Is the processor process running?
+
+    * Log into processor node and watch the processor logs for errors.
+
+      FIXME: List log paths.
+
+    * Check datadog ``processor.save_raw_and_processed`` for appropriate
+      environment.
+
+Is the processor saving to S3?
+
+    * Check datadog
+      ``processor.s3.BotoS3CrashStorage.save_raw_and_processed`` for
+      appropriate environment.
+
+Is the processor saving to ES?
+
+    * FIXME: Check datadog?
+
+Is the processor saving to Postgres?
+
+    * Check datadog
+      ``processor.postgres.PostgreSQLCrashStorage.save_raw_and_processed`` for
+      appropriate environment.
+
+Submit a crash. Verify processed crash made it to S3, ES and postgres.
+
+
+Webapp
+======
+
+Is the webapp up?
+
+    * Use a browser and check the healthcheck.
+
+      :prod: https://crash-stats.mozilla.com/monitoring/healthcheck/
+      :stage: https://crash-stats.allizom.org/monitoring/healthcheck/
+
+      It should say "ok: true".
+
+Is the webapp throwing errors?
+
+    * Check sentry for errors
+    * Log into webapp node and check logs for errors.
+
+      FIXME: List log paths.
+
+    * Run QA Selenium tests.
+
+      :prod: ``webqatestbot build socorro.prod.saucelabs``
+      :stage: ``webqatestbot build socorro.stage.saucelabs``
+
+Can we log into the webapp?
+
+    * Log in.
+    * FIXME: What is behind a login that we need to verify is working?
```

Comment:
The profile page is not "important" but it is indeed the only page that is only accessible to logged in users, whatever their permissions or role. 

It is indeed important to be able to access restricted resources once you're logged in with the right permissions, but I don't know what the best way to test that would be. Do we test with a superuser, that can access everything? Do we test with a user that only has the most common permissions? What do we test? That's a lot of questions, and I guess we could just go with the simplest choice. All these features are unit tested anyway.